app/src/main/java/chat/rocket/android/layouthelper/ExtRealmModelListAdapter.java

```diff
@@ -37,6 +41,35 @@ protected void notifyRealmModelItemChanged(int position) {
     notifyItemChanged(position + 1);
   }
 
+  public void setLayoutManager(RecyclerView.LayoutManager layoutManager) {
+    this.layoutManager = layoutManager;
+  }
+
+  protected ListUpdateCallback listUpdateCallback = new ListUpdateCallback() {
+    @Override
+    public void onInserted(int position, int count) {
+      notifyItemRangeInserted(position + 1, count);
+      if (layoutManager != null) {
+        layoutManager.scrollToPosition(0);
+      }
```

Comment:
I suspect this causes unexpected scroll.
When we read very past message, it is better not to force scroll to the bottom by new message arrival.

Also, I think it is better for this kind of functionality to be implemented with RealmListObserver, not with passing layouthelper like this.

```
latestMessageObserver =  realmHelper
        .createObjectObserver(
            realm -> realm.where(Message.class).equalTo("rid", roomId))
        .setImpl(results -> results.last(null))
        .setOnUpdateListener(this::scrollToBottomIfNeeded);
```

in RoomFragment.